### DISCUSSION TITLE
Page unauthorized

### DISCUSSION THREAD

**Discussion by FilipVysin** [requarks/wiki#5542] [Error / Bug Report]
Hello, we have a problem with a page that is visible to everyone but unregistered users cannot access it, I tried to redo it but the same problem

**Comment by NGPixel** [comment]
Make sure the Guests group has read access to that path.

**Comment by FilipVysin** [comment]
Screen https://ctrlv.cz/Yxdm

**Reply by NGPixel** [reply]
That's not how permissions work. Please read https://docs.requarks.io/groups.

What you pointed in the screenshot only specifies who can see **the menu item**, not **the actual page** being referenced. You need to set the proper page rule in the Guests group.

**Comment by FilipVysin** [comment]
screen https://ctrlv.cz/1E7v
